src/accelerate/commands/launch.py

```diff
@@ -303,6 +303,16 @@ def launch_command_parser(subparsers=None):
         type=str,
         help="Tee std streams into a log file and also to console.",
     )
+    distributed_args.add_argument(
+        "--log-dir",
```

Comment:
No need to have this one, our args are a bit special and will let this happen automatically :) 
```suggestion
```

Reply:
Sure, I deleted that line, thanks for the suggestion!